content/german/components/rfid.md

```diff
@@ -0,0 +1,109 @@
+---
+title: RFID
+tags: ["spi"]
+---
+
+## Funktionsweise
+
+Bei RFID, abgekürzt für Radio Frequency Identification, handelt es sich um eine Technologie, welche es ermöglicht automatisch und kontaktlos
+mit einer Karte entsprechende Informationen auszutauschen. Dies wird zum Beispiel bei weit bekannten NFC Tags verwendet, jedoch auch für
+Anwendungen wie das kontaktlose Zahlen per Kreditkarte.
+
+Ein grosser Vorteil ist hierbei, dass die jeweiligen Karten/Tags, auch Transponder genannt, so klein wie ein Reiskorn sein können und keine
+eigene Stromversorgung benötigen. Im CrowPi ist die Komponente `MFRC522` verbaut, welche ein hochfrequentes elektronisches Wechselfeld bei
+13.56 MHz erzeugt, um damit einerseits die Karte mit Strom zu versorgen und andererseits mit dieser zu kommunizieren. Somit können
+beispielsweise die beiden beigelegten Tags des CrowPi ausgelesen und beschrieben werden mit beliebigen Daten.
+
+Die Technologie dahinter ist sehr komplex und die entsprechenden Informationen dazu sind auf mehrere Standards aufgeteilt, hauptsächlich
+`ISO-14443` für den allgemein Aufbau und die Kommunikation sowie die entsprechenden Spezifikationen von NXP Semiconductors, dem Hersteller
```

Comment:
Do you have a link to ISO-14443?

Reply:
I've made some slight changes to the docs, what are your thoughts on the updated version?